### PR TITLE
Create database

### PR DESCRIPTION
**Pull request by hakichris**
In this branch, we create a schema file for the blog app database. 

- [x] no linters error 
- [x] schema added 
- [x]  Gitflow

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,166 @@
...
-* Database creation
+<!-- PROJECT DESCRIPTION -->
 
-* Database initialization
+# 📖 [Blog] <a name="about-project"></a>
 
-* How to run the test suite
 
-* Services (job queues, cache servers, search engines, etc.)
+**[OOP: SchooL library]** A fully functional website that will show the list of posts and empower readers to interact with them by adding comments and liking posts
```
Comment (anagudelogu) [inline README.md:35]: - [ ] Please consider updating your project information to match the Blog project, at the moment it has the description for OOP School library. 🤓 🔧

File: README.md [hunk 2]
```diff
@@ -1,24 +1,166 @@
...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/hakichris/OOP-School-Library/issues).
```
Comment (anagudelogu) [inline README.md:140]: - [ ] Also please consider fixing this link as currently, it redirects users to OOP-School-Library issues. 🤓 🔧

**Review by anagudelogu — CHANGES_REQUESTED** [review]
# STATUS:  CHANGES REQUIRED  ♻️ 🚧 

Hi @hakichris, 
I'm Andres.


While you have done a great job working on this milestone 💻 , there are some issues that you still need to work on to go to the next project. ♻️ 

You are almost there, keep up the good work! 🚀 



## Highlights
✅ Great use of Git flow. 💯 
✅ Your project is documented professionally. 📜 
✅ Descriptive PR title, summary and commit messages. 💬 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (hakichris) [reply README.md:35]: fixed

File: README.md (hunk 2 above)
Comment (hakichris) [reply README.md:140]: fixed, thank you

**Review by ridwanediallo — APPROVED** [review]
# STATUS: APPROVED 🟢
## Hi @hakichris , 
Congratulations!GIF

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- You used GitHub flow correctly. ✔
- You implemented the requested changes successfully. ✔

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**